src/egui/mod.rs

```diff
@@ -364,7 +367,7 @@ impl UpdatesApp {
             ui.add_enabled_ui(!self.v.serial_query.is_empty() && self.v.search_promise.is_none(), | ui | {
                 if !input_submitted && !ui.button("Search for updates").clicked() { return; }
 
-                let already_searched = self.v.update_results.iter().any(|e| e.title_id == parse_title_id(&self.v.serial_query));
+                let already_searched = self.v.update_results.iter().any(|e| e.title_id == crate::psn::parse_title_id(&self.v.serial_query));
```

Comment:
Shouldn't `parse_title_id` be available already with the `use::crate::psn::*`?

Reply:
Correct, seems that my IDE added this when I was experimenting with modules. Done